### PR TITLE
bridge_ui: Make transfers to/from chains configurable

### PR DESCRIPTION
**Pull request by kev1n-peters**
This change is [Reviewable](https://reviewable.io/reviews/certusone/wormhole/997)

### CODE REVIEW

File: bridge_ui/src/config.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  warningMessage?: WarningMessage;
+}
+
+export type ChainConfigMap = {
+  [key in ChainId]?: ChainConfig;
+};
+
+export const CHAIN_CONFIG_MAP: ChainConfigMap = {};
+
+CHAIN_CONFIG_MAP[CHAIN_ID_POLYGON] = {
```
Comment (kev1n-peters) [inline bridge_ui/src/config.ts:24]: type checking doesn't work on this object's properties when it's included in the initialization of CHAIN_CONFIG_MAP...

Comment (evan-gray) [reply bridge_ui/src/config.ts]: It will if you use `as ChainConfigMap` after :wink:

Comment (kev1n-peters) [reply bridge_ui/src/config.ts]: Changed

**Review by evan-gray — APPROVED** [review]
:lgtm:

Was wondering if the warning message should show regardless of the disabled state, but I think I agree with the way it is. The message can be written in a way to convey the status, and folks probably want to know if the opposite way is disabled ahead of time. In the future may want to consider it showing as an error if the selected direction is disabled.

> Reviewed 5 of 7 files at r1, 1 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/certusone/wormhole/997#-:-MyK5XJID1v7PHbjxiIt:b-1s8ypx)* status: 6 of 7 files reviewed, 1 unresolved discussion (waiting on @chase-45 and @kev1n-peters)

**Review by chase-45 — APPROVED** [review]

**Pull request merged**